app/views/index/normal.phtml

```diff
@@ -138,4 +134,11 @@ $today = @strtotime('today');
 </main>
 <?php endif; ?>
 
+<?php $class = $this->displaySlider ? ' active' : ''; ?>
+<a href="#" id="close-slider" class="<?= $class ?>">
+	<?= _i('close') ?>
+</a>
+<div id="slider" class="scrollbar-thin<?= $class ?>">
```

Comment:
I like the improvement, that it is outside of `<main>`
Would be `<aside>` the better semantic tag, while it is now outside of `<main>`?

Reply:
Sounds like a good idea

Reply:
This required a few more changes https://github.com/FreshRSS/FreshRSS/pull/4375/commits/33b3605e473f3cc0032cf622b51df3e0858f2e41
Tests welcome